### DISCUSSION TITLE
Esta semana en Rust #20

### DISCUSSION THREAD

**Discussion by giscus** [RustLangES/blog#48] [Comentarios de Artículos]
# Esta semana en Rust #20

El crate de esta semana es efs, una implementación del sistema de archivos ext2 sin estándar con planes para agregar otros sistemas de archivos en el futuro. - By @

https://www.rustlang-es.org/blog/articles/2024-05-01-this-week-in-rust.html

**Comment by mariannegoldin** [comment]
¡Muchas gracias por brindar esta versión en español de This Week in Rust a la comunidad! Lo apreciamo. - Marianne, coeditora de TWIR
